playbooks/openshift-metrics/private/config.yml

```diff
@@ -25,7 +25,7 @@
     import_role:
       name: openshift_metrics
       tasks_from: update_master_config.yaml
-    when: not openshift.common.version_gte_3_9
+    when: not openshift.common.version_gte_3_9 and (openshift_metrics_install_metrics | bool)
```

Comment:
openshift.common.version_gte_3_9 should be true for all 3.9 installs so maybe this PR is not necessary?

Reply:
I tried uninstalling metrics without this patch and it worked fine.

Seems that this task should be removed in 3.9